src/NHSD.BuyingCatalogue.Documents.API/Program.cs

```diff
@@ -14,7 +16,17 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
+                    webBuilder.UseConfiguration(GetConfiguration());
```

Comment:
@assoomro `Host.CreateDefaultBuilder(args)` applies defaults (see [remarks](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webhost.createdefaultbuilder?view=aspnetcore-3.1#Microsoft_AspNetCore_WebHost_CreateDefaultBuilder_System_String___)) that include all of those currently in the `GetConfiguration` method, so we should be able to remove this line and `GetConfiguration` without affecting the application.

Reply:
That is correct, will get rid of it.